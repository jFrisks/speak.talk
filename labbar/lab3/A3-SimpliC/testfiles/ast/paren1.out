Program
  List
    FunctionDecl
      IdDecl ID="a"
      List
      Block
        List
          IdDeclStmt
            IdDecl ID="x"
          Assignment
            IdUse ID="x"
            Mul
              Sub
                Numeral NUMERAL="3"
                Numeral NUMERAL="4"
              Numeral NUMERAL="5"
