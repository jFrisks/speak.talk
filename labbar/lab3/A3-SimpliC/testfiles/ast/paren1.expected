Program
  List
    FunctionDecl ID="a"
      List
      Block
        List
          IdDecl ID="x"
          Assignment
            IdUse ID="x"
            Mul
              Sub
                Numeral NUMERAL="3"
                Numeral NUMERAL="4"
              Numeral NUMERAL="5"